lib/controllers/issueish-detail-controller.js

```diff
@@ -290,23 +209,37 @@ export class BareIssueishDetailController extends React.Component {
     await this.props.workspace.open(uri, {pending: true});
     addEvent('open-commit-in-pane', {package: 'github', from: this.constructor.name});
   }
+
+  openReviews = async () => {
```

Comment:
we have very similar methods for opening reviews in the `IssueishDetailController` and the `IssueishListController`.  Would be good to consolidate those, to prevent the kind of bugs in the future where the URIs diverge and we open multiple tabs of the same thing.

Reply:
`ReviewsItem.buildURI()` now accepts an options object :+1:

I'm not sure what other cleanup we could do here. :thinking: Unresolve if you have suggestions :smile: